src/pages/ContentScripts/features/community-openrank-racing-bar/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Container, render } from 'react-dom';
+import $ from 'jquery';
+
+import features from '../../../../feature-manager';
+import isPerceptor from '../../../../helpers/is-perceptor';
+import { getRepoName, isPublicRepoWithMeta } from '../../../../helpers/get-repo-info';
+import { getOpenRank } from '../../../../api/community';
+import { getActivityDetails } from '../../../../api/repo';
+import View from './view';
+import DataNotFound from '../repo-networks/DataNotFound';
+import { CommunityOpenRankDetails } from './data';
+import { JsonObject } from 'type-fest';
+
```

Comment:
Some imported items are not used in the subsequent code, and similar situations in the code can be removed.

Reply:
@wangyantong2000 Thanks for detailed review. ❤️